tests/unit_tests/init.js

```diff
@@ -1014,5 +1020,73 @@ describe('init tests', () => {
 
       fs.unlinkSync('test_config.conf.js');
     });
+
+    test('generateConfig with js with allowAnonymousMetrics', () => {
+      mockery.registerMock(
+        './logger',
+        class {
+          static error() {}
+        }
+      );
+
+      mockery.registerMock(
+        'uuid',
+
+        class {
+          static v4() {
+            return '3141-5926-5358-9793';
+          }
+        }
+      );
+
+      const answers = {
+        language: 'js',
+        backend: 'local',
+        browsers: ['chrome', 'firefox'],
+        defaultBrowser: 'firefox',
+        allowAnonymousMetrics: false
+      };
+
+      const {NightwatchInit} = require('../../lib/init');
+      const nightwatchInit = new NightwatchInit(rootDir, []);
```

Comment:
Can we also add an assertion after this that `nightwatchInit.client_id` is set as expected?

Reply:
updated.